backend/modules/pdf_processing/pipeline.py

```
@@ -0,0 +1,363 @@
+            })
+            
+            total_time = time.time() - start_time
+            debug_log(f"[PIPELINE] Full processing completed in {total_time:.2f}s")
+            
+            return {
+                "success": True,
+                "data": ai_result,
+                "session_id": session_id,
+                "scenario_id": scenario_id,
+                "message": "PDF parsed successfully"
+            }
```

Ruff (0.14.5)

Consider moving this statement to an `else` block

(TRY300)